powerauth-push-server/src/main/java/io/getlime/push/service/ApnsPayloadBuilder.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2023 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.getlime.push.service;
+
+import com.eatthepath.pushy.apns.DeliveryPriority;
+import com.eatthepath.pushy.apns.util.SimpleApnsPayloadBuilder;
+import com.google.firebase.messaging.ApnsConfig;
+import com.google.firebase.messaging.Aps;
+import com.google.firebase.messaging.ApsAlert;
+import io.getlime.push.model.entity.PushMessageBody;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Convert {@link PushMessageBody} to APNs platform-dependent payload.
+ *
+ * @author Lubos Racansky, lubos.racansky@wultra.com
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+@Slf4j
+final class ApnsPayloadBuilder {
+
+    private ApnsPayloadBuilder() {
+        throw new IllegalStateException("Should not be instantiated.");
```

Comment:
I know that I am the author of this class 😅 But today, I think that _builder_ is not a perfect name. I mean I am surprised by the hidden constructor, I would expect that `ApnsPayloadBuilder` builds `ApnsPayload` like `...builder().build()`, but this class is more a convertor from `ApnsPayloadBuilder` to APNs JSON payload, isn't it?

Reply:
I will rename the class.

Reply:
Thanks!